### PR TITLE
Disable for Version, Delete Button | Added time format for Table Last Run | CSS fixes

### PR DESCRIPTION
**Pull request by oumkale**
## Proposed changes

Disable for Version, Delete Button | Added time format for Table Last Run | CSS fixes

## Types of changes
Screenshot from 2020-10-14 16-08-43
Screenshot from 2020-10-14 16-08-40

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/components/Targets/ClusterInfo/index.tsx [hunk 1]
```diff
@@ -75,8 +75,8 @@ const ClusterInfo: React.FC<ClusterVarsProps> = ({ location }) => {
                 </div>
               </div>
               <div className={classes.expDiv}>
-                {/* Set isDisabled to false when implemented */}
-                <ButtonOutline handleClick={() => {}} isDisabled>
+                {/*
+                <ButtonOutline handleClick={() => {}} isDisabled={false}>
```
Comment (Saranya-jena) [inline litmus-portal/frontend/src/components/Targets/ClusterInfo/index.tsx:79]: Button is already disbaled, you don't need to remove it completely.

**Review by Saranya-jena — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/components/Targets/ClusterInfo/index.tsx (hunk 1 above)
Comment (oumkale) [reply litmus-portal/frontend/src/components/Targets/ClusterInfo/index.tsx:79]: I dissabled that now, we need this for next release.

Comment (oumkale) [reply litmus-portal/frontend/src/components/Targets/ClusterInfo/index.tsx]: PTAL again @Saranya-jena

**Review by ishangupta-ds — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]
LGTM. @A4k9 Can you take a look? The button has been removed for now.

**Review by A4k9 — APPROVED** [review]

**Pull request merged**